src/Jackett.Common/Indexers/ApiBay.cs

```diff
@@ -0,0 +1,224 @@
+            AddCategoryMapping(499, TorznabCatType.ConsoleOther, "Games Other");
+            // Porn
+            AddCategoryMapping(500, TorznabCatType.XXX, "Porn");
+            AddCategoryMapping(501, TorznabCatType.XXX, "Movies");
+            AddCategoryMapping(502, TorznabCatType.XXXDVD, "Movies DVDR");
+            AddCategoryMapping(503, TorznabCatType.XXXImageset, "Pictures");
+            AddCategoryMapping(504, TorznabCatType.XXX, "Games");
+            AddCategoryMapping(505, TorznabCatType.XXX, "HD - Movies");
+            AddCategoryMapping(506, TorznabCatType.XXX, "Movie Clips");
+            AddCategoryMapping(599, TorznabCatType.XXXOther, "Porn other");
+            // Other
+            AddCategoryMapping(600, TorznabCatType.Other, "Other");
+            AddCategoryMapping(601, TorznabCatType.Books, "E-books");
+            AddCategoryMapping(602, TorznabCatType.BooksComics, "Comics");
+            AddCategoryMapping(603, TorznabCatType.Books, "Pictures");
+            AddCategoryMapping(604, TorznabCatType.Books, "Covers");
+            AddCategoryMapping(605, TorznabCatType.Books, "Physibles");
+            AddCategoryMapping(699, TorznabCatType.BooksOther, "Other Other");
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+
+            var releases = await PerformQuery(new TorznabQuery());
+
+            await ConfigureIfOK(
+                string.Empty,
+                releases.Any(),
+                () => throw new Exception("Could not find releases from this URL")
+                );
+
+            return IndexerConfigurationStatus.Completed;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var categories = MapTorznabCapsToTrackers(query);
+
+            var queryStringCategories = string.Join(
+                ",",
+                categories.Count == 0
+                    ? GetAllTrackerCategories()
+                    : categories
+                );
+
+            var response = await RequestWithCookiesAsync(
+                $"{SiteLink}q.php?q={query.SearchTerm}&cat={queryStringCategories}"
+                );
+
+            return JsonConvert
+                .DeserializeObject<List<QueryResponseItem>>(response.ContentString)
+                .Select(CreateReleaseInfo);
+        }
+
+        private ReleaseInfo CreateReleaseInfo(QueryResponseItem item)
+        {
+            var magnetUri = new Uri(MagnetUri.Replace(KeyInfoHash, item.InfoHash));
+
+            return new ReleaseInfo
+            {
+                Title = item.Name,
+                Category = MapTrackerCatToNewznab(item.Category.ToString()),
+                MagnetUri = magnetUri,
+                InfoHash = item.InfoHash,
+                PublishDate = ParseUtil.ParseDateTimeFromUnixEpochTimeStamp(item.Added),
```

Comment:
I believe routine ParseDateTimeFromUnixEpochTimeStamp is a duplicate of an already existing tool, as shown in this example
https://github.com/Jackett/Jackett/blob/621a47338dcd98ff0b2471b7df99f7995e10a4dd/src/Jackett.Common/Indexers/HDBitsApi.cs#L119
please note I'm not a c# dev so you'll need to verify for yourself

Reply:
This appears to produce the same results in my testing. I've removed the duplicate method in favor of this one. Thanks!